source/slang/slang-ir-strip.h

```diff
@@ -0,0 +1,18 @@
+// slang-ir-strip.h
+#pragma once
+
+namespace Slang
+{
+    struct IRModule;
+
+    struct IRStripOptions
+    {
+        bool shouldStripNameHints = false;
+    };
+
+        /// Strip out instructions that should only be used by the front-end.
+    void stripFrontEndOnlyInstructions(
+        IRModule*               module,
+        IRStripOptions const&   options);
+}
+#pragma once
```

Comment:
What's with the double #pragma once?

Reply:
Probably a copy-paste issue. I'd resolve it now but that would mean waiting for CI to run yet again. :(